src/modules/reporting/components/reporting-dispute-progress/reporting-dispute-progress.jsx

```diff
@@ -13,7 +13,8 @@ const ReportingDisputeProgress = (p) => {
   let repStakedFormatted = formatAttoRep(p.stakeCurrent, { decimals: 4, roundUp: true }).formatted
 
   if (userStaked) {
-    userPercentage = calculateAddedStakePercentage(bondSizeCurrent.fullPrecision, p.accountStakeCurrent, p.tentativeStake)
+    const accountStakeCurrentFormatted = (p.accountStakeCurrent > 0 ? formatAttoRep(p.accountStakeCurrent, { decimals: 4, roundUp: true }).formatted : 0)
+    userPercentage = calculateAddedStakePercentage(bondSizeCurrent.fullPrecision, accountStakeCurrentFormatted, p.tentativeStake)
```

Comment:
Need to check disputing with over 1000 REP because formatted will give you ',' and calculatedAddedStakePercentage will blow up. might need to use formattedValue